client/src/pages/login/mainLogin/login.jsx

```diff
@@ -10,21 +10,22 @@ class Login extends React.Component {
 			  	<article className="login">
 			  		<h1>This is the login screen</h1>   {/* Big words of "this is the login screen" in middle of page */}
 			  	</article>
-			  	<nav className = "link">
+			  	<nav className = "login-link">
```

Comment:
Remove this section because we will have a form that will run direct an authenticated user to the correct page. For testing we will leave the pages open so we can just goto them via the url
```
<nav className = "login-link">
			  		<Link to = '/applicant'>Applicant</Link>   {/* React Router link to Applicants Dash */}
			  	</nav>
			  	<nav className = "link">
			  	<nav className = "login-link">
			  		<Link to = '/volunteer'>Volunteer</Link>   {/* React Router link to Volunteers Dash */}
			  	</nav>
			  	<nav className = "link">
			  	<nav className = "login-link">
			  		<Link to = '/judge'>Judge</Link> 	   {/* React Router link to Judges Dash */}
			  	</nav>
			  	<nav className = "link">
			  	<nav className = "login-link">
			  		<Link to = '/admin'>Admin</Link>	   {/* React Router link to Admins Dash */}
			  	</nav>
```

Reply:
Deleted that specific content, and changed `/#event` to `/event`